src/aiida/orm/utils/builders/computer.py

```
@@ -102,12 +102,10 @@ def new(self):
                 mpiprocs_per_machine = int(mpiprocs_per_machine)
             except ValueError:
                 raise self.ComputerValidationError(
-                    'Invalid value provided for mpiprocs_per_machine, ' 'must be a valid integer'
+                    'Invalid value provided for mpiprocs_per_machine, must be a valid integer'
                 )
```

Ruff (0.15.21)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)